module/VuFind/src/VuFind/ChannelProvider/ChannelLoader.php

```diff
@@ -127,8 +157,8 @@ protected function getChannelsFromResults($providers, Results $results, $token)
      */
     protected function getChannelProviders($source, $configSection, $activeId = null)
     {
-        $providerIds = isset($this->config->{"source.$source"}->$configSection)
-            ? $this->config->{"source.$source"}->$configSection->toArray() : [];
+        $providerIds = isset($this->config["source.$source"][$configSection])
+            ? $this->config["source.$source"][$configSection] : [];
```

Comment:
This can be simplified now:


```suggestion
        $providerIds = $this->config["source.$source"][$configSection] ?? [];
```